crates/wasmparser/src/readers/core/globals.rs

```diff
@@ -37,13 +37,15 @@ impl<'a> FromReader<'a> for Global<'a> {
 
 impl<'a> FromReader<'a> for GlobalType {
     fn from_reader(reader: &mut BinaryReader<'a>) -> Result<Self> {
+        let content_type = reader.read()?;
+        let flags = reader.read_u8()?;
+        if flags > 0b11 {
+            bail!(reader.original_position() - 1, "malformed mutability")
```

Comment:
Mind updating this to a different error message along the lines of "malformed global flags byte" or similar?

(might as well go ahead and implement it as-if shared everything was stable)